examples/js/effects/VREffect.js

```diff
@@ -399,12 +399,12 @@ THREE.VREffect = function ( renderer, onError ) {
 
 			return;
```

Comment:
I believe the `else` is unnecessary here as you've a `return` if presenting or am I missing something here?